### PR TITLE
Adds Topology as top-level nav item

### LINKED ISSUE #3349: Add Topology as top-level nav item
Towards the ongoing effort https://github.com/patternfly/patternfly-extension-issues/issues/11

Per discussion with @jeff-phillips-18 , @dgutride , and the rest of the extensions team, "Topology" will be moved out of "Extensions" and added as a top-level nav item in the workspace.

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #3349 , dependent on https://github.com/patternfly/patternfly-react/pull/8508

### How to test:

In order to see the topology demos under the "Topology" section, you need to make [this change](https://github.com/patternfly/patternfly-react/pull/8586/files) in org under `node_modules/@patternfly/react-topology/src/components/TopologyView/examples/Topology.md`  and then rebuild.

### CODE REVIEW

**Review by jenny-s51 — COMMENTED** [review]
WIP. Investigating missing css styles... ws topology styles are not getting pulled in. Marking as draft until this is resolved

**Review by jenny-s51 — COMMENTED** [review]
Added missing CSS imports to `packages/v4/patternfly-docs/patternfly-docs.css.js` - these imports resolve the topology styling issues in the org workspace.

File: packages/v4/patternfly-docs/patternfly-docs.source.js [hunk 1]
```diff
@@ -76,6 +80,9 @@ module.exports = (sourceMD, sourceProps) => {
   sourceMD(path.join(logViewerContentBase, '/**/examples/*.md'), 'react');
   sourceMD(path.join(logViewerContentBase, '/**/demos/*.md'), 'react-demos');
 
+  // React-topology MD
+  sourceMD(path.join(reactTopologyPath, '/**/components/**/examples/*.md'), 'react');
```
Comment (jeff-phillips-18) [inline packages/v4/patternfly-docs/patternfly-docs.source.js:84]:
```suggestion
  sourceMD(path.join(reactTopologyPath, '/**/examples/*.md'), 'react');
```
I'm thinking we'll end up wanting docs for things outside of components as well

**Review by jeff-phillips-18 — DISMISSED** [review]

**Review by wise-king-sullyman — DISMISSED** [review]

**Review by evwilkin — DISMISSED** [review]
LGTM!

**Review by evwilkin — APPROVED** [review]

**Review by jeff-phillips-18 — APPROVED** [review]

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**